test/cmd.js

```diff
@@ -467,6 +506,60 @@ describe('express(1)', function () {
     })
   })
 
+  describe('--no-view', function () {
+    var ctx = setupTestEnvironment(this.fullTitle())
+
+    it('should create basic app with no view engine', function (done) {
+      run(ctx.dir, ['--view', 'none'], function (err, stdout) {
```

Comment:
It looks like the command that is being executed does not match the description. I thought you removed --view=none in favor of --no-view. Why does the old command still work?